pkg/controller/nodenetworkconfigurationpolicy/conditions/enactments.go

```diff
@@ -130,3 +130,30 @@ func setEnactmentMatching(conditions *nmstatev1alpha1.ConditionList, message str
 		message,
 	)
 }
+
+type CountByConditionType map[nmstatev1alpha1.ConditionType]int
+
+func enactmentsCount(enactments nmstatev1alpha1.NodeNetworkConfigurationEnactmentList) CountByConditionType {
+	trueConditionsFrequency := CountByConditionType{}
```

Comment:
Again, not a frequency IMO, what about just countByConfigurationType?

Reply:
Fixing.